allennlp/models/bert_for_classification.py

```diff
@@ -0,0 +1,141 @@
+from typing import Dict, Union
+
+from overrides import overrides
+import torch
+from pytorch_pretrained_bert.modeling import BertModel
+
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.model import Model
+from allennlp.nn.initializers import InitializerApplicator
+from allennlp.training.metrics import CategoricalAccuracy
+
+
+@Model.register("bert_for_classification")
+class BertForClassification(Model):
+    """
+    An AllenNLP Model that runs pretrained BERT,
+    takes the pooled output, and adds a Linear layer on top.
+    If you want an easy way to use BERT for classification, this is it.
+    Note that this is a somewhat non-AllenNLP-ish model architecture,
+    in the sense that it goes straight from indexed tokens to pooled output
```

Comment:
I would say _why_ it goes against our philosophy, not _how_ it goes against it.  That is, "it's not configurable", not, "it doesn't use these abstractions".